pkg/restclient/examples/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gluster/glusterd2/restclient"
```

Comment:
As you may already know, this is causing the CI failure. It has to be `github.com/gluster/glusterd2/pkg/restclient`